### PR TITLE
Product and regular product form in modal

### PR DESCRIPTION
**Pull request by AlexFry123**

### CODE REVIEW

File: src/app/domains/Product/components/combined/ProductCombinedForm/ProductCombinedForm.template.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import { useState } from 'react'
+import { Modal } from 'app/components/Lib/Modal'
+import { IconButton } from '@material-ui/core'
+import AddCircleIcon from '@material-ui/icons/AddCircle'
+import { ProductSimpleForm } from '../../forms/ProductSimpleForm'
+import { useForm } from 'mui-form-generator-fractal-band-2'
+import { Switch } from '@material-ui/core'
+import { RegularProductSimpleForm } from '../../../../RegularProduct/components/forms/RegularProductSimpleForm'
```
Comment (okrdima) [inline src/app/domains/Product/components/combined/ProductCombinedForm/ProductCombinedForm.template.js:8]: fix import

File: src/app/domains/Product/components/combined/ProductCombinedForm/ProductCombinedForm.template.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import { useState } from 'react'
+import { Modal } from 'app/components/Lib/Modal'
+import { IconButton } from '@material-ui/core'
+import AddCircleIcon from '@material-ui/icons/AddCircle'
+import { ProductSimpleForm } from '../../forms/ProductSimpleForm'
```
Comment (okrdima) [inline src/app/domains/Product/components/combined/ProductCombinedForm/ProductCombinedForm.template.js:5]: fix import

File: src/app/domains/RegularProduct/components/forms/RegularProductSimpleForm/RegularProductSimpleForm.template.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import {
+  Form,
+  FormGenerator,
+  FormButtons
+} from 'mui-form-generator-fractal-band-2'
+import Button from '@material-ui/core/Button'
+import RegularProductSimpleSelect from '../../select/RegularProductSingleSelect/RegularProductSingleSelect.template'
```
Comment (okrdima) [inline src/app/domains/RegularProduct/components/forms/RegularProductSimpleForm/RegularProductSimpleForm.template.js:7]: fix import

File: src/app/domains/RegularProduct/components/forms/RegularProductSimpleForm/RegularProductSimpleForm.template.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import {
+  Form,
+  FormGenerator,
+  FormButtons
+} from 'mui-form-generator-fractal-band-2'
+import Button from '@material-ui/core/Button'
+import RegularProductSingleSelect from 'app/domains/RegularProduct/components/select/RegularProductSingleSelect'
```
Comment (GingerBoy14) [inline src/app/domains/RegularProduct/components/forms/RegularProductSimpleForm/RegularProductSimpleForm.template.js:7]: fix import

File: src/app/domains/Product/components/forms/ProductSimpleForm/ProductSimpleForm.template.js [hunk 5]
```diff
@@ -1,5 +1,52 @@
...
 const ProductSimpleForm = (props) => {
-  return <div>ProductSimpleForm</div>
+  const { onSubmit, onSubmitFail, formProps, form } = props
+  return (
+    <Form
+      form={form}
+      onSubmit={onSubmit}
+      onSubmitFail={onSubmitFail}
+      {...formProps}>
+      <FormGenerator config={config} show={show} />
```
Comment (GingerBoy14) [inline src/app/domains/Product/components/forms/ProductSimpleForm/ProductSimpleForm.template.js:46]: How to pass `show` property to this form?

File: src/app/domains/Product/components/combined/ProductCombinedForm/ProductCombinedForm.template.js [hunk 6]
```diff
@@ -0,0 +1,80 @@
+import { useState } from 'react'
+import { Modal } from 'app/components/Lib/Modal'
+import { IconButton } from '@material-ui/core'
+import AddCircleIcon from '@material-ui/icons/AddCircle'
+import { ProductSimpleForm } from 'app/domains/Product/components/forms/ProductSimpleForm'
+import { useForm } from 'mui-form-generator-fractal-band-2'
+import { Switch } from '@material-ui/core'
+import { RegularProductSimpleForm } from 'app/domains/RegularProduct/components/forms/RegularProductSimpleForm'
+import { Row } from '@qonsoll/react-design'
+import { Typography } from '@material-ui/core'
```
Comment (GingerBoy14) [inline src/app/domains/Product/components/combined/ProductCombinedForm/ProductCombinedForm.template.js:10]: Use one m-ui import `.../core` instead 3, fix another imports

File: src/app/domains/RegularProduct/components/forms/RegularProductSimpleForm/RegularProductSimpleForm.template.js [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+  return (
+    <Form
+      form={form}
+      onSubmit={onSubmit}
+      onSubmitFail={onSubmitFail}
+      {...formProps}>
+      <FormGenerator config={config} show={show} />
+      <FormButtons Button={Button} visible={false} />
+    </Form>
+  )
```
Comment (GingerBoy14) [inline src/app/domains/RegularProduct/components/forms/RegularProductSimpleForm/RegularProductSimpleForm.template.js:43]: How to pass `show` property to this form?

**Review by GingerBoy14 — CHANGES_REQUESTED** [review]

File: src/app/domains/Product/components/combined/ProductCombinedForm/ProductCombinedForm.template.js [hunk 8]
```diff
@@ -1,13 +1,11 @@
 import { useState } from 'react'
-import { Modal } from 'app/components/Lib/Modal'
-import { IconButton } from '@material-ui/core'
+import { IconButton, Switch, Typography } from '@material-ui/core'
 import AddCircleIcon from '@material-ui/icons/AddCircle'
-import { ProductSimpleForm } from 'app/domains/Product/components/forms/ProductSimpleForm'
+import { Row } from '@qonsoll/react-design'
 import { useForm } from 'mui-form-generator-fractal-band-2'
-import { Switch } from '@material-ui/core'
+import { Modal } from 'app/components/Lib/Modal'
```
Comment (sashka2131) [inline src/app/domains/Product/components/combined/ProductCombinedForm/ProductCombinedForm.template.js:6]: Enough to use only 'app/components/Lib ' there main index wich export this component

**Review by sashka2131 — APPROVED** [review]

**Review by GingerBoy14 — APPROVED** [review]

**Pull request merged**